PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/RuleConfigLoader.java

```diff
@@ -209,6 +217,7 @@ private Map<String, String> getMapOfDiscouragedIdentifiers(JsonReader reader, Ma
 
             switch (fieldName) {
                 case "methods_to_check":
+                case "clients_to_check":
```

Comment:
nit: Do we do the same handling for methods_to_check vs client to check?

Reply:
yes they'll both be keys in the map